python/dlr/__init__.py

```diff
@@ -6,7 +6,8 @@
 from __future__ import absolute_import as _abs
 
 from .api import DLRModel
+from .metadata import NAME, VERSION
 
-__all__ = ['DLRModel']
+__all__ = NAME
 
-__version__ = "1.3.0"
+__version__ = VERSION
```

Comment:
Why move this?

Reply:
This is because of cyclical dependency conflicts when setup tool calling __init__.py, which then invoke singleton phone-home that requires setup tool to install the third party dependencies. This is a way to bypass that